python.md

```diff
@@ -0,0 +1,87 @@
+# Use faster BLAS routines for SciPy and NumPy
+
+The default SciPy and NumPy binary installations with `pip3 install numpy scipy`
+are configured to use OpenBLAS. On Ubuntu `apt install python3-numpy
+python3-scipy` will install NumPy and SciPy with BLAS and LAPACK libraries.
+The default installations of SciPy and NumPy are easy to setup and well
+tested.
+
+Some workloads will benefit from using BLIS. Benchmarking SciPy and NumPy
+workloads with BLIS might allow to identify additional performance improvement.
+If NumPy and SciPy detect the presence of the BLIS library, they will use BLIS
+in priority over the same functionality from BLAS and OpenBLAS.
+OpenBLAS or LAPACK libraries need to be installed along BLIS to provide LAPACK
+functionality.
+
+To install the prerequisites to build SciPy and NumPy with BLIS and OpenBLAS on
+Ubuntu and Debian:
+```
+sudo apt -y install build-essential python3-pip libopenblas-dev gfortran libblis-dev
+```
+
+Prerequisites to build SciPy and NumPy with BLIS on ARM64 AL2 and RedHat:
+```
+# Install AL2/RedHat prerequisites
+sudo yum install "@Development tools" python3-pip python3-devel blas-devel gcc-gfortran
+
+# Install BLIS
+git clone https://github.com/flame/blis $HOME/blis
+cd $HOME/blis;  ./configure --enable-threading=openmp --enable-cblas cortexa57
+make -j4;  sudo make install
+
+# Install OpenBLAS
+git clone https://github.com/xianyi/OpenBLAS.git $HOME/OpenBLAS
+cd $HOME/OpenBLAS
+make -j4 BINARY=64 FC=gfortran USE_THREAD=1
+sudo make PREFIX=/usr/local install
+
+# Set dynamic linker's path to installed libraries
+export LD_LIBRARY_PATH=/usr/local/lib
+```
+
+To build and install NumPy and SciPy with BLIS and OpenBLAS:
+```
+git clone https://github.com/numpy/numpy/ $HOME/numpy
+cd $HOME/numpy;  pip3 install .
+
+git clone https://github.com/scipy/scipy/ $HOME/scipy
+cd $HOME/scipy;  pip3 install .
+```
+
+BLIS and OpenBLAS will be printed as library dependences:
```

Comment:
how do you chose? which one is preferred by default?

Reply:
I added a paragraph that describes the order in which NumPy and SciPy will choose the BLAS and LAPACK functionality when there are multiple providers of the same interface.